packages/client/libp2pBrowserBuild/util/index.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * @module util
+ */
+import { bytesToPrefixedHexString } from '@ethereumjs/util'
```

Comment:
we can use client's util, it has browser handling, no reason to duplicate i think